src/Components/TodoItem.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => (
```

can be a bit simplify